lib/administrate/fields/password.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "base"
+
+module Administrate
+  module Field
+    class Password < Field::Base
+      def self.searchable?
+        false
+      end
+
+      def truncate
```

Comment:
I'm not sure anyone would have a reason to pass in a truncation length. It seems like we could either display a fixed number of asterisks if the password exists, or an empty string if the password doesn't exist. Thoughts?